exotica_python/test/test_no_unknown_initializer_types.py

```diff
@@ -1,8 +1,24 @@
 from __future__ import print_function
+import os
 import subprocess
+import unittest
 
-output = subprocess.check_output("python -c 'import pyexotica as exo ; initializers = exo.Setup.get_initializers()'", shell=True)
 
-if "Skipping" in output:
-    print(output)
-    assert "Skipping" not in output
+class TestUnknownInitializerTypes(unittest.TestCase):
+    def test_unknown_initializer_types(self):
+        output = None
+        if os.environ['ROS_PYTHON_VERSION'] == '2':
+            output = subprocess.check_output(
+                "/usr/bin/env python -c 'import pyexotica as exo\ninitializers = exo.Setup.get_initializers()'", shell=True)
+        elif os.environ['ROS_PYTHON_VERSION'] == '3':
+            output = subprocess.check_output(
+                "/usr/bin/env python3 -c 'import pyexotica as exo\ninitializers = exo.Setup.get_initializers()'", shell=True)
```

Comment:
Codacy found an issue: [subprocess call with shell=True seems safe, but may be changed in the future, consider rewriting without shell](https://app.codacy.com/gh/ipab-slmc/exotica/pullRequest?prid=5739305)